app/bundles/FormBundle/Tests/EventListener/ReportSubscriberTest.php

```diff
@@ -129,30 +156,103 @@ public function testOnReportBuilderAddsFormAndFormSubmissionReports(): void
                 $setGraphs[] = $args;
             });
 
+        $this->companyReportData->expects($this->once())
+            ->method('getCompanyData')
+            ->with()
+            ->willReturn([]);
+
+        $this->subscriber->onReportBuilder($mockEvent);
+
+        $this->assertCount(2, $setTables);
+        $this->assertCount(3, $setGraphs);
+    }
+
+    public function testOnReportBuilderAddsFormAndFormResultReports(): void
+    {
+        $mockEvent = $this->getMockBuilder(ReportBuilderEvent::class)
+            ->disableOriginalConstructor()
+            ->onlyMethods([
+                'checkContext',
+                'getStandardColumns',
+                'getCategoryColumns',
+                'addTable',
+            ])
+            ->getMock();
+
+        $mockEvent->expects($this->once())
+            ->method('getStandardColumns')
+            ->willReturn([]);
+
+        $mockEvent->expects($this->once())
+            ->method('getCategoryColumns')
+            ->willReturn([]);
+
+        $mockEvent->expects($this->exactly(3))
+            ->method('checkContext')
+            ->willReturnOnConsecutiveCalls(true, false, true);
+
         $form = $this->getMockBuilder(Form::class)
             ->disableOriginalConstructor()
             ->onlyMethods([
                 'getFields',
+                'getMappedFieldValues',
             ])
             ->getMock();
 
+        $field = $this->createMock(Field::class);
+        $field->expects($this->exactly(2))
+            ->method('getAlias')
+            ->willReturn('email');
+        $field->expects($this->once())
+            ->method('getLabel')
+            ->willReturn('Email');
+        $field->expects($this->exactly(3))
+            ->method('getType')
+            ->willReturn('string');
+
         $form->expects($this->once())
             ->method('getFields')
-            ->willReturn([]);
+            ->willReturn(new ArrayCollection([$field]));
+
+        $form->expects($this->once())
+            ->method('getMappedFieldValues')
+            ->willReturn([
+                [
+                    'idFormFields' => 1,
```

Comment:
The "s" seem to be a typo

Reply:
Oh, yes.
I replaced the dto mocking with object creation.  And this code was removed.